src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy

```diff
@@ -52,32 +52,30 @@ class PublishContent extends AbstractCommand {
 		if (publishingTarget != 'live' && publishingTarget != 'staging') {
 			throw new CommandLine.ParameterException(commandSpec.commandLine(), 'Invalid publishing target. Use live or staging')
 		}
+		if (!title) {
+			throw new CommandLine.ParameterException(commandSpec.commandLine(), 'Missing required option title')
```

Comment:
I think should have a default message here. (If no default, then the param must be mandatory. But in this case, this should be defaulted to make it easier to use.) The default can be dynamic and based on some information we have from the user.